### PR TITLE
Added a CRC hash get function for helping Job Avoision

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala [hunk 1]
```diff
@@ -264,7 +267,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   }
 
   /**
-   * Takes two arrays of WDL Files and generates any necessary JES inputs from them.
+   * Takes two arrays of remote and local WDL File paths and generates the necessary JESInput's.
```
Comment (cjllanwarne) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: Unrelated, but this comment was awkward before.

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala [hunk 2]
```diff
@@ -159,6 +158,10 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     }
   }
 
+  def getCrc32c(googleCloudStoragePath: GoogleCloudStoragePath): String = authenticated { implicit connection =>
+    authenticated { _.storage.getCrc32c(googleCloudStoragePath) }
```
Comment (Horneth) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: Why double `auhenticated` ?

**Comment by geoffjentry** [comment]
Is this scheme robust to large files?

**Comment by cjllanwarne** [comment]
@geoffjentry According to the Google documentation, yes. This is the only method which is applicable to composite objects.

**Comment by geoffjentry** [comment]
Perfect, thanks

File: src/main/scala/cromwell/util/google/GoogleCloudStorage.scala [hunk 3]
```diff
@@ -36,6 +36,11 @@ case class GoogleCloudStorage(client: Storage) {
     new GcsBucketInfo(bucketName, bucket.getLocation, bucket.getTimeCreated, bucket.getOwner)
   }
 
+  def getCrc32c(googleCloudStoragePath: GoogleCloudStoragePath): String = {
```
Comment (mcovarr) [inline src/main/scala/cromwell/util/google/GoogleCloudStorage.scala]: Could you add a comment that addresses @geoffjentry's question, maybe with a link to the relevant GCS docs?

**Comment by mcovarr** [comment]
:+1:

**Comment by Horneth** [comment]
:+1:

**Comment by cjllanwarne** [comment]
Closing because I'll merge into job_avoidance instead.

**Pull request closed without merging**